modules/nf-core/openms/idfilter/main.nf

```diff
@@ -0,0 +1,57 @@
+process OPENMS_IDFILTER {
```

Comment:
Maybe outside of this PR, but doesnt this module also need to have an environment.yml?

Reply:
These modules have not been ported to use nf-test, they still use pytest. On the agenda though😁